cime_config/machines/config_machines.xml

```diff
@@ -1153,7 +1152,7 @@
 
     <environment_variables compiler="craygnu.*" mpilib="mpich">
       <env name="PKG_CONFIG_PATH">/lustre/orion/cli115/world-shared/frontier/3rdparty/protobuf/21.6/gcc-native-13.2/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}</env>
-      <env name="ADIOS2_ROOT">$SHELL{if [ -z "$ADIOS2_ROOT" ]; then echo /lustre/orion/cli115/world-shared/frontier/3rdparty/adios2/2.10.2/cray-mpich-8.1.30/craygnuamdgpu/cpe-24.07; else echo "$ADIOS2_ROOT"; fi}</env>
+      <env name="ADIOS2_ROOT">$SHELL{if [ -z "$ADIOS2_ROOT" ]; then echo /lustre/orion/cli115/world-shared/frontier/3rdparty/adios2/2.10/install/craygnu-hipcc/cpe-24.11; else echo "$ADIOS2_ROOT"; fi}</env>
```

Comment:
@trey-ornl Does the old ADIOS2_ROOT cause any build errors? If not, we can keep it. P.S. I plan to rebuild the ADIOS2 libraries with the latest modules on Frontier and update ADIOS2_ROOT to use them in a separate PR later.

Reply:
I reverted the path change and ran an NE30 test successfully. I committed the un-change.